_posts/2018-09-28-3D-Printing-Intro.md

```diff
@@ -0,0 +1,400 @@
+
+<div class="responsive-video">
+    <iframe width="1164" height="655" src="https://www.youtube.com/embed/GUdnrtnjT5Q" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+
+<br/>
+
+#### FOOD
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/3D-print-food.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+Food printing has been a hype for a long time.
+Culinary experts have been experimenting with the concept, but also companies like NASA for astronauts.
+
+Simple items like chocolate and dough can already be 3D printed, based on the FDM method, the edible object is printed layer by layer.
+There are 'robots' that can print more complex items, but these are not actual 3D printers and more automated assembly machines, the boundary between 3D printing and assembly can become a bit blurry.
+At this point in time it is not yet possible to print entire meals from raw base ingredients.
+This will no doubt be developed further as time progresses.
+
+
+## Getting started with 3D printing
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/3D-print-fail.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+Getting started with 3D printing yourself is not easy, it can be very daunting.
+There are a lot of things you need to take into account
+
+Since it is still relatively expensive, you want to ask yourself these questions:
+
+- What type of 3D printing do I want?
+- What is the best (price/quality) printer available?
+- Will I use it enough to justify the purchase?
+- You could also just order a printed model online!
+
+Getting things right is hard:
+
+- Printer calibration
+- Model slicing & print settings
+
+⇒ 3D printing remains an intensive trial and error process!
+Only begin if you are willing to invest the necessary time into it.
+If at first you don't succeed, try again, and again, and again, and again!
+
+Thankfully the online hobby community is very large and generally very helpful.
+You will need to model your object, or download it from a maker website.
+You will need to slice your model with setting for your specific printer, and adjust these settings based on the quality and speed you want.
+
+*Online resources:*
+
+- Youtube channels:
+  - [Maker's Muse](https://www.youtube.com/user/TheMakersMuse)
+  - [3D Printing Nerd](https://www.youtube.com/channel/UC_7aK9PpYTqt08ERh1MewlQ)
+  - [Make Anything](https://www.youtube.com/channel/UCVc6AHfGw9b2zOE_ZGfmsnw)
+- Forums:
+  - [3D Hubs](https://www.3dhubs.com/)
+  - [Reddit 3D printing](https://www.reddit.com/r/3Dprinting/)
+
+  These YouTube channels and forums are an invaluable resource when getting into 3D printing, they contains loads of information, do's and don'ts.
+  By watching and reading videos and articles you can prevent mistakes and gain insights faster in how 3D printing works and what is possible.
```

Comment:
by watching and reading videos and articles => by watching videos and reading articles

but still a sentence with a lot of 'ands'